assets/src/dashboard/app/views/editorSettings/publisherLogo/test/publisherLogo.js

```diff
@@ -115,123 +115,10 @@ describe('PublisherLogo', () => {
         isLoading={false}
         publisherLogos={formattedPublisherLogos}
         uploadError={'Something went wrong.'}
-        canUploadFiles={true}
+        canUploadFiles
       />
     );
 
     expect(getByText('Something went wrong.')).toBeInTheDocument();
   });
-
-  it('should trigger mockHandleRemoveLogo when delete button is pressed with enter on an uploaded file', () => {
-    const mockHandleRemoveLogo = jest.fn();
-
-    const { getByTestId } = renderWithProviders(
-      <PublisherLogoSettings
-        handleAddLogos={jest.fn}
-        handleRemoveLogo={mockHandleRemoveLogo}
-        handleUpdateDefaultLogo={mockHandleRemoveLogo}
-        isLoading={false}
-        publisherLogos={formattedPublisherLogos}
-      />
-    );
-
-    const ContextMenuButton = getByTestId(
-      'publisher-logo-context-menu-button-2'
-    );
-    fireEvent.click(ContextMenuButton);
```

Comment:
**Part 3**

After fixing the first two bugs, these tests were now failing! When `fireEvent.click` was called on the button, the focusout event was called for that specific publisher logo. This is super weird, and I'm not sure why this happened. However, the karma tests _continued_ to work, so I figured it must be something strange with the `useFocusOut` hook and the react testing library.

To deal with this, I wrote new karma tests to cover these tests. I removed these tests since they were no longer needed.